bindings/python/iota_sdk/utils.py

```diff
@@ -218,7 +218,7 @@ def verify_secp256k1_ecdsa_signature(
 
     @staticmethod
     def verify_transaction_semantic(
-            inputs: transaction: Transaction, inputs: List[InputSigningData], unlocks: Optional[List[Unlock]] = None) -> str:
+            transaction: Transaction, inputs: List[InputSigningData], unlocks: Optional[List[Unlock]] = None) -> str:
```

Comment:
Missing protocol_parameters

Reply:
hm that may be missing at more places, dangit

Reply:
or not :D

Reply:
You only added it to signing?